firefox-ios/firefox-ios-tests/Tests/ClientTests/Messaging/NimbusMessagingMessageTests.swift

```diff
@@ -12,6 +12,16 @@ import Shared
 @testable import Client
 
 final class NimbusMessagingMessageTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+        DependencyHelperMock().bootstrapDependencies()
```

Comment:
needed to add the dependency mocks to resolve this error
![Screenshot 2024-04-24 at 11 22 00 AM](https://github.com/mozilla-mobile/firefox-ios/assets/6743397/46074863-e526-4592-8953-edfad7fe104f)